content/chainguard/chainguard-enforce/vulnerabilities/vuln-reports-and-attestations.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Vulnerability reports and Attestations in Chainguard Enforce"
+linktitle: "Vulnerability reports and Attestations"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/vuln-reports-and-attestations/
+- /chainguard/chainguard-enforce/concepts/vuln-reports-and-attestations/
+type: "article"
+description: "An overview of how Chainguard Enforce handles attestations of vulnerability reports"
+date: 2023-03-19T15:56:52-07:00
+lastmod: 2023-03-19T15:56:52-07:00
+draft: false
+tags: ["ENFORCE", "COSIGN", "VULNERABILITIES", "CONCEPTUAL"]
+images: []
+menu:
+  docs:
+    parent: "concepts"
+weight: 030
+toc: true
+---
+
+In order to fully leverage the capabilities offered by Chainguard Enforce, it's important to establish a process for generating quality vulnerability reports.
+A vulnerability report lists all the discovered vulnerabilities contained in your software. It uses a [SBOM](/open-source/sbom/) to search for vulnerabilities matching the used packages in your software.
+Chainguard Enforce can capture vulnerabilities data and use it to help you find all the vulnerabilities of the packages in your ecosystem, as well as apply and enforce security policies to your runtime environments.
```

Comment:
```suggestion
Chainguard Enforce can ingest vulnerability reports and use their data to help you find vulnerabilities in  packages within your application's software ecosystem. It can also apply and enforce security policies for your runtime environments that take vulnerability data into account.
```

I'm not 100% sure about that last `take vulnerability data into account` part!